site/_posts/2020-03-18-Milestones_Cadence_Kasparov.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Milestones, Release Cadence, Kasparov
+author: chessbyte
+date: 2020-03-18 13:33:33 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+ManageIQ is getting ready to release an alpha version of the Jansa release.  Additionally, we are moving to a faster release cadence - approximately once every 3 months to keep up with the rate of change of all the providers that ManageIQ interacts with.  We hope to release Jansa GA by June and the K-release by September.  You will notice new milestones in the ManageIQ community to help us keep track of the features being planned for a particular release.  The milestones and the [roadmap](https://github.com/orgs/ManageIQ/projects/13) should help keep us all on track!
+
+At this point in time, we usually offer the community an opportunity to vote on the next release name.  An exception is being made to the name of the K-release to honor one of my all-time favorite world chess champions - Garry Kasparov.  
```

Comment:
Can we link Kasparov's name? `[Garry Kasparov](https://en.wikipedia.org/wiki/Garry_Kasparov)`

Reply:
done!